### ISSUE TITLE
Some shortcuts don't work 

### ISSUE THREAD

**Issue by McFaul4** [8bitPit/Niagara-Issues#2172]
### Describe the bug

When I add shortcuts to some specific apps, they can't be used/opened

### Steps to reproduce

1. Sesame/Niagara Web Search Shortcuts
---Active the Niagara web search or sesame 
---Search a Website or therm
a) With Niagara web search you can long hold the search Suggestion
b) with sesame you have to click search via a Browser or Google (you can add them in sesame settings). Afterwards the Search result is displayed if you search the same thing. Long hold it
--- add it to your favorites/show it in your App list
---it won't open

2. The settings shortcuts
---edit Favorites
---add shortcuts
---tap more
---add one of them (I've used BT and battery saver)
---it won't open

Seems like they aren't the only ones: it's the same with WhatsApp shortcuts to open a chat directly 

### Does the issue also occur with other 3rd-party launchers?

Problem doesn't occur with other launchers

### Other 3rd-party launcher(s) tested

Nova 7 prime 

### Device name

OnePlus Nord

### Android version

Android 12

### Screenshots / Screen recordings

![Screenshot_2023-01-11-17-31-48-61_a4824db61f40c53807964ea8b14c92fe](https://user-images.githubusercontent.com/116670013/211869879-2b660fd5-8115-418a-9f0b-edc65acf1bb4.jpg)

**Comment by Amorphous404** [comment]
Hey, Thanks for reporting, Does it show any error or states anything when you tap/open the added shortcuts?

Also can you please try to update to the latest release and check if you can reproduce the issue:
https://github.com/8bitPit/Niagara-Issues/releases/tag/v1.8.11

**Comment by McFaul4** [comment]
Nope, just shows the Animation of the click. And the new version didn't fix the bug either..

**Comment by Amorphous404** [comment]
Does restarting the Launcher or Phone help?

**Comment by McFaul4** [comment]
Nope, tried both. (And updated the launcher)

**Comment by Amorphous404** [comment]
https://github.com/8bitPit/Niagara-Issues/issues/1930

**Comment by McFaul4** [comment]
So I can't get a Sesame Shortcut to my Home screen on Microsoft launcher and the settings shortcuts don't work over there. 

But like I said before, everything works in the Nova Launcher.
